### PR TITLE
feat: [Novo] Convert more routes

### PR DESCRIPTION
**Pull request by damassi**
Addresses: https://artsyproduct.atlassian.net/browse/PLATFORM-3115 

This converts `/consign`, `/debug` and `/example`. (And /consign route has been added to Calibre.)

### CODE REVIEW

File: src/v2/Apps/getAppNovoRoutes.tsx [hunk 1]
```diff
@@ -36,6 +38,9 @@ export function getAppNovoRoutes(): RouteConfig[] {
       {
         routes: artworkRoutes,
       },
+      {
+        routes: auctionsRoutes,
+      },
```
Comment (joeyAghion) [inline src/v2/Apps/getAppNovoRoutes.tsx:43]: Should this route be reflected in `webpackPublicPath` too (or only later when `/auctions2` replaces `/auctions`)?

Comment (icirellik) [reply src/v2/Apps/getAppNovoRoutes.tsx:43]: It doesn't look like this was `converted: true`, seems like an unrelated change to keep the routes in sync.

Comment (damassi) [reply src/v2/Apps/getAppNovoRoutes.tsx:43]: Yep 👍

File: src/desktop/lib/webpackPublicPath.ts [hunk 2]
```diff
@@ -31,6 +31,9 @@ if (process.env.NODE_ENV === "production") {
     "/collect",
     "/collection",
     "/collections",
+    "/consign",
```
Comment (icirellik) [inline src/desktop/lib/webpackPublicPath.ts:34]: It doesn't look like consign was converted...?

Comment (damassi) [reply src/desktop/lib/webpackPublicPath.ts:34]: Ah, foggy head on Friday -- fixed.

**Comment by icirellik** [comment]
Going to make sure we didn't import something crazy into the bundle before this is merge.

**Comment by icirellik** [comment]
This PR increases the total bundle size by 7%, I would like to do some further investigation to see why. It has to be related to the example app or auctions.

**Review by icirellik — APPROVED** [review]

**Pull request merged**